src/components/ProjectList.tsx

```diff
@@ -60,10 +81,12 @@ const ProjectList: React.FC = () => {
   const handleProjectClick = (project: Project) => {
     setSelectedProject(project);
     setOpenProjectDialog(true);
+    navigate(`/#${project.name.toLowerCase().replace(/\s/g, '-')}`);
```

Comment:
Maybe we should use folder names for the slug instead of `project.name.toLowerCase().replace(/\s/g, '-')`. Folder name is less likely to change, and is guaranteed to be unique.

e.g. [projects/example-cipher-c10l-hello-world/description.yaml#L2](https://github.com/oasisprotocol/playground/blob/ff98542ae05209ee59c386d5fa2871c60408b958/projects/example-cipher-c10l-hello-world/description.yaml#L2):
`example-cipher-c10l-hello-world` vs `confidential-hello-world-on-cipher`

cc @matevz